massa-node/base_config/config.toml

```diff
@@ -8,7 +8,7 @@
     # max number of future periods considered during requests
     draw_lookahead_period_count = 10
     # port on which the node API listens for admin and node management requests. Dangerous if publicly exposed. Bind to "[::1]:port" if you want to access the node from IPv6.
-    bind_private = "127.0.0.1:33034"
+    bind_private = "0.0.0.0:33034"
```

Comment:
Why did you change this? Isn't listening only on localhost a protection we want to keep?
If it is needed, also change the comment to [::]:port for IPv6 instructions

Reply:
good catch :), it was a mistake

Reply:
I did the changes for all similar lines to be concise